src/runner.ts

```diff
@@ -178,4 +181,70 @@ export class StaticAnalysisParserRunner {
 
         return undefined;
     }
+
+    private async readConvertedReport(reportPath: string): Promise<types.ReportContents> {
+        const reportContent = await fs.promises.readFile(reportPath, 'utf8');
+        return JSON.parse(reportContent);
+    }
+
+    private parseConvertedReport(reportContent: types.ReportContents): types.VulnerabilityDetail[] {
+        logger.debug(messagesFormatter.format(messages.parsing_converted_report));
+
+        const severityMap = {
+            'note': 'LOW',
+            'warning': 'MEDIUM',
+            'error': 'HIGH',
+            'critical': 'CRITICAL'
+        };
+
+        const rules: Record<string, types.Rule> = this.getRules(reportContent);
+        const vulnerabilities: types.VulnerabilityDetail[] = reportContent.runs[0].results.map(result => ({
+            external_id: uuidv4(),
```

Comment:
Can we use the ID from our report?
And we need to generate an ID if there is no ID in report, refer to https://github.com/parasoft/parasoft-findings-vsts/blob/master/PublishParasoftResults/StaticAnalysisReportService.ts#L165.